Contracts/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "dap.ps-contracts",
+  "version": "1.0.0",
+  "description": "Embark contracts",
+  "main": "index.js",
+  "scripts": {
+    "build:testnet": "embark build testnet",
+    "build:development": "embark build development",
+    "build": "embark build livenet",
```

Comment:
Same story here. `embark` is not in `PATH`. Use `../node_modules/.bin/embark`.

Reply:
Noted will amend

Reply:
As my previous comment, my understand is that when commands are run from within the npm script that it resolves the node modules of the local code base inclusively to the global commands